### PR TITLE
feature/action-bar

### PR DESCRIPTION
**Pull request by RafaelGondi**
#### What is the purpose of this pull request?
Creates the ActionBar component

#### How should this be manually tested?
* Test using the actions addon and the knobs addon;
* Try to change the background color of the component;
* Check the isDismissible checkbox and see if an 'x' icon is displayed;
* See if an action is triggered when the 'close' icon is pressed;
* See if the ActinoBar is displayed floating and in the bottom of the page
when the Floating Mode is setted to true;

#### Screenshots or example usage
image

#### Types of changes
- [x]  New feature (a change which adds functionality)
- [ ] Hot fix (a change which fixes an issue found in the master branch)
- [ ]  Documentation change

### CODE REVIEW

File: src/components/ActionBar.vue [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	align-items: center;
+}
+
+.fixed {
+	position: fixed;
+}
+
+.icon-container {
+	padding: 2px 4px;
+    background-color: #0000002d;
```
Comment (esron) [inline src/components/ActionBar.vue]: Nessa linha a indentação está com espaços.

```suggestion
	background-color: #0000002d;
```

Comment (esron) [reply src/components/ActionBar.vue]: Verificar o resto do arquivo.

File: test/unit/ActionBar.spec.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	const wrapper = mount(ActionBar, {
+		localVue,
+		propsData: {
+			isDismissible: true,
+		},
+	});
+
+	wrapper.find('.icon-container').trigger('click');
+
+	expect(wrapper.emitted().close).toBeTruthy();
```
Comment (esron) [inline test/unit/ActionBar.spec.js:47]: Qual a diferença da linha 47 pra linha 48? Em termos do que está sendo testado.

**Review by esron — CHANGES_REQUESTED** [review]

**Review by esron — APPROVED** [review]

File: test/unit/ActionBar.spec.js (hunk 2 above)
Comment (RafaelGondi) [reply test/unit/ActionBar.spec.js:47]: The line 47 asserts that the event has been emitted properly and the line 48 asserts that the value it emitted is equal to'true'

**Pull request merged**